components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5607,9 +5621,24 @@ private boolean isMultivaluedAttribute(String name) {
      * @param attribute
      */
     private void populateBasicAttributes(LocalClaim mappedLocalClaim, AbstractAttribute attribute, boolean
-            isEnterpriseExtensionAttr) {
+            isEnterpriseExtensionAttr, boolean isCustomSchemaAttr) {
+
+        boolean isMultivaluedInAdditionalProp = false;
+        boolean isBooleanInAdditionalProp = false;
 
         if (mappedLocalClaim != null) {
+            if (mappedLocalClaim.getClaimProperties() != null) {
+                for (Map.Entry<String, String> claimProperty : mappedLocalClaim.getClaimProperties().entrySet()) {
+                    if (StringUtils.equalsIgnoreCase(claimProperty.getKey(), SCIMConfigConstants.MULTIVALUED)) {
```

Comment:
Use the pattern `<Constant>.equalIgnoreCase(<variable>)`

Reply:
Resolved in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/459/commits/a227f2e3af8b1ad2afaaab992d44c5f8e2b64c70